app/viewmodels/TradeJourneyStatusesViewModel.scala

```diff
@@ -223,8 +227,15 @@ object TradeJourneyStatusesViewModel {
             .url,
           expenses.construction.routes.ConstructionIndustryCYAController.onPageLoad(taxYear, businessId).url
         )
-      case ExpensesInterest | ExpensesTotal | NationalInsurance | TradeDetails | ExpensesAdvertisingOrMarketing | ExpensesDepreciation |
-          ExpensesProfessionalFees | ExpensesOtherExpenses =>
+      case ExpensesDepreciation =>
+        determineUrl(
+          expenses.depreciation.routes.DepreciationDisallowableAmountController
+            .onPageLoad(taxYear, businessId, NormalMode)
+            .url,
```

Comment:
DepreciationDisallowableAmount is the first page (usually it would be an allowable amount page). The journey only has one amount page which asks for the disallowable amount.